hcx/hcxlicensingoptions.md

```diff
@@ -0,0 +1,38 @@
+# VMware HCX Licensing Options
+
+HCX comes in 2 different licensing versions:
+1.	VMware HCX Advanced
```

Comment:
Advanced is no longer available in AVS.

Reply:
I will remove it. However, the following page has the same issue:
https://learn.microsoft.com/en-us/azure/azure-vmware/install-vmware-hcx#hcx-license-edition

If we are saying "by default" everyone gets an "Enterprise" license. We also need to update the above page.

Reply:
Need 100% confirmation on this as Advanced is still deployed by default (not sure if it's a region thing) with the button to self upgrade to Enterprise. I've changed the wording to reflect the latter until get 100% confirmation.